src/eShop.AppHost/Program.cs

```diff
@@ -81,6 +81,24 @@
     builder.AddOpenAI(catalogApi, webApp);
 }
 
+bool useOllama = false;
+if (useOllama)
+{
+    var ollama = builder.AddOllama("ollama")
+        .WithDataVolume()
+        .WithGPUSupport()
+        .WithOpenWebUI();
+    var embeddings = ollama.AddModel("embedding", "all-minilm");
+    var chat = ollama.AddModel("chat", "llama3.1");
+
+    catalogApi.WithReference(embeddings)
+        .WithEnvironment("OllamaEnabled", "true")
+        .WaitFor(embeddings);
+    webApp.WithReference(chat)
+        .WithEnvironment("OllamaEnabled", "true")
+        .WaitFor(chat);
```

Comment:
Can we put this in Extensions.cs like the `AddOpenAI` method is?

Reply:
Done.